test/scope-key-data-test.js

```diff
@@ -147,3 +147,18 @@ testHelpers.dev.devOnlyTest("Warn when key is not found and log the value of the
 	scopeKeyData.read();
 	teardown();
 });
+
+testHelpers.dev.devOnlyTest("Don't warn when key is not defined #206", function(assert) {
+	var teardown = testHelpers.dev.willWarn(/Unable to find key "foo.length". Found "foo" with value: %o/, function(message, matched) {
```

Comment:
I think this should just check that the warning `"Unable to find key "foo.length".` was displayed. Use a string instead of the regex so it has to match exactly.

Reply:
Updated, just this case doesn't log a warning.

Reply:
@phillipskevin I updated the test, thank you!